tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -79,55 +79,55 @@ Docker logs backward compatibility
     Should Contain  ${output}  container ${id1} does not support '--timestamps'
 
 Docker logs with tail
-    ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create busybox sh -c 'seq 1 5000'
+    ${rc}  ${id}=  Run And Return Rc And Output  docker1.11 %{VCH-PARAMS} create busybox sh -c 'seq 1 5000'
```

Comment:
I think all tests in this suite should be using docker client 1.13, now that the VCH has been upgraded.